slides.md

```diff
@@ -156,57 +156,99 @@ import anotherFunction, {aFunction} from "./someFile";
 
 ---
 # Hello Todo!
-1. Edit `App.js` so that the list rendered from an array:
+Currently, `App.js` is quite boring, it would be great if instead of having the todos hardcoded, we could somehow cycle through an array of todos and show a `Text` per element.
```

Comment:
Nit: here and elsewhere, `todos` -> `TODOs`

Reply:
grammarly suggested to-do, should I use to-dos? 👀

Reply:
I would stick with `TODO` and `TODOs` because `to-do` seems to have a different meaning outside of programming context ([sounds posh to me](https://www.google.co.uk/search?q=define+to-do&oq=define+to-do&aqs=chrome..69i57j0l5.3377j1j1&sourceid=chrome&ie=UTF-8))

Reply:
9bc52a5